README.md

```diff
@@ -157,6 +157,10 @@ class Page
 
   # default from a singleton value (boolean in this case)
   attribute :published, Boolean, :default => false
+  
+  # default from a singleton value (string in this case),
+  # that doesn't allow a nil value
+  attribute :author, String, :default => 'Mies', :allow_nil => false
```

Comment:
I assume `page = Page.new(title: nil, author: nil)` and `page = Page.new` would both do the same thing, right? The latter seems better to include in the examples.